### PR TITLE
fix: ensure disable culling for avatar preview

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

**Review by sandrade-dcl — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs [hunk 1]
```diff
@@ -191,4 +201,13 @@ public void Rotate(float rotationVelocity)
     {
         avatarRenderer.transform.Rotate(Time.deltaTime * rotationVelocity * Vector3.up);
     }
+
+    private void EnforceDisableAvatarCulling()
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs:205]: We can't just stop the `CullingController` before taking the snapshot and start it again afterwards? When stopping it, it ensures all the renderers go back to the initial state. I think `SetCullingForRenderer` shouldn't be exposed from outside like this as its breaks encapsulation.

**Review by BrianAmadori — CHANGES_REQUESTED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs:205]: I think that currently stopping CullingController doesn't reset the renderers to the initial state.
I can add that, but there are two possible scenarios there:
1- resetting the all renderers without using a coroutine means that we might have a hiccup there when taking snapshot
2- resetting renderers inside a coroutine means that clicking "save" in avatar editor will take longer to process and close the editor (which I think that already takes too long)

**Review by pravusjif — APPROVED** [review]

**Comment by pravusjif** [comment]
Tested with a guest user and saw no problem when generating the local snapshots

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs (hunk 1 above)
Comment (BrianAmadori) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs:205]: Resetting all the renderers will involve a pretty negligible time compared to taking the snapshot and sending it over to kernel, so I think we can do it safely.

**Comment by pbosio** [comment]
I'm changing the approach on another PR

**Pull request closed without merging**